setup.py

```diff
@@ -13,18 +12,7 @@ def _read_requirements_file():
 
 class build_ext(_build_ext.build_ext):
     def run(self):
-        try:
-            import tensorflow
-        except ImportError:
-            subprocess.check_call(['pip', 'install', 'tensorflow>=0.11.0'])
-
-        try:
-            import gym
-        except ImportError:
-            subprocess.check_call(
-                ['pip', 'install',
-                 'git+https://github.com/openai/gym.git@'
-                 '93d554bdbb4b2d29ff1a685158dbde93b36e3801#egg=gym'])
+        pass
```

Comment:
Mmm, why are we getting rid of a tensorflow/gym install?

Reply:
it was a pep8 thing, and I figured the requirements have it covered. we can add `flake8: noqa` if we need to keep as if though

Reply:
undid the changes and added the ignore